app/metrics.yaml

```diff
@@ -23,3 +23,152 @@ distribution:
     notification_emails:
       - dmu@mozilla.com
     expires: "2020-05-01"
+pages:
+  page_load:
+    type: timing_distribution
+    lifetime: application
+    description: >
+      Counting how long each page takes to load.
+    send_in_pings:
+      - metrics
```

Comment:
This is redundant, these metrics will go by default into the `metrics ping`. You can remove both lines.